File: cashroulette_integrated_notification_sample/example-cashroulette-java/src/main/java/com/example/example_cashroulette_java/PreferenceUtil.java
package com.example.example_cashroulette_java;

import android.content.Context;
import android.content.SharedPreferences;

public class PreferenceUtil {
    private static final String PREFERENCE_NAME = "notification_prefs";
    private Context context;
    private SharedPreferences prefs;
    private SharedPreferences.Editor editor;

    public PreferenceUtil(final Context context) {
        this.context = context;
        this.prefs = this.context.getSharedPreferences(PREFERENCE_NAME, Context.MODE_PRIVATE);
        this.editor = prefs.edit();
    }

    public Boolean getChecked(final String key, final Boolean defValue) {
        return prefs.getBoolean(key, defValue);

    }

    public void setChecked(final String key, final Boolean value) {
        editor.putBoolean(key, value).apply();
    }
}

File: cashroulette_integrated_notification_sample/example-cashroulette-kotlin/src/main/java/com/example/integrated_notification_cashroulette/PreferenceUtil.kt
package com.example.integrated_notification_cashroulette

import android.content.Context
import android.content.SharedPreferences

class PreferenceUtil(context: Context) {
    private val prefs: SharedPreferences = context.getSharedPreferences("notibar_prefs", Context.MODE_PRIVATE)

    fun getChecked(key: String, defValue: Boolean = false): Boolean {
        return prefs.getBoolean(key, defValue)
    }

    fun setChecked(key: String, value: Boolean) {
        prefs.edit().putBoolean(key, value).apply()
    }
}